tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -1568,6 +1641,11 @@ def test_post_bulk_creates_job_and_dispatches_celery_task(
     if is_scheduled:
         scheduled_for = datetime.utcnow() + timedelta(days=1)
         data["scheduled_for"] = scheduled_for.isoformat()
+    if has_default_reply_to:
+        create_reply_to_email(sample_email_template.service, "test@test.com")
+    if use_sender_id:
+        reply_to_email = create_reply_to_email(sample_email_template.service, "custom@test.com", is_default=False)
+        data["reply_to_id"] = reply_to_email.id
```

Comment:
It's nice to read the tests sometimes and getting more what we support as features.